azure-pipelines.yml

```diff
@@ -296,6 +296,14 @@ jobs:
         testGroup: outerloop
         readyToRun: true
 
+# Format
```

Comment:
I don't think the formatting job should be run unconditionally. 

I think the condition should be 
```yaml
and(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest', 'IndividualCI', 'BatchedCI'), eq(variables['Build.DefinitionName'], 'coreclr-ci'))
```